.github/workflows/conventional-commit.yml

```diff
@@ -0,0 +1,25 @@
+name: Conventional Commit Check
+
+on: [pull_request]
+
+jobs:
+  conventional-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check
```

Comment:
There is a commit linter available in the GH marketplace https://github.com/marketplace/actions/commit-linter, it might be easier to use an existing action, rather than composing our own regex, thoughts?